### PR TITLE
Update Shaka Player to include compiled CSS

### PR DESCRIPTION
**Pull request by joeyparrish**
This updates the npm file map to include compiled CSS files which are necessary to use the UI build of the library released in v2.5.

## Essential checklist

* [x] I'm the author of this library
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

### CODE REVIEW

**Comment by joeyparrish** [comment]
@PeterDaveHello, @MattIPv4, could someone please review this change?

**Comment by joeyparrish** [comment]
Is this project still alive and accepting updates?

**Comment by MattIPv4** [comment]
We're currently working to refactor how the service works to enable updates to be reviewed more easily, see https://github.com/cdnjs/cdnjs/issues/13652 & https://github.com/cdnjs/cdnjs/issues/13613

File: ajax/libs/shaka-player/package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     {
       "basePath": "dist",
       "files": [
-        "shaka-player*"
+        "shaka-player*",
+        "controls.css*"
```
Comment (MattIPv4) [inline ajax/libs/shaka-player/package.json]: Are `demo.*` or `receiver.*` files needed? Or just `shaka-player*` and `controls.css*`?

```
ls node_modules/shaka-player/dist

controls.css                           demo.compiled.map                      receiver.compiled.debug.js             shaka-player.compiled.debug.js         shaka-player.ui.debug.externs.js       shaka-player.ui.map
controls.css.map                       demo.css                               receiver.compiled.debug.map            shaka-player.compiled.debug.map        shaka-player.ui.debug.js               wrapper.js
demo.compiled.debug.js                 demo.css.map                           receiver.compiled.js                   shaka-player.compiled.externs.js       shaka-player.ui.debug.map
demo.compiled.debug.map                deps.js                                receiver.compiled.map                  shaka-player.compiled.js               shaka-player.ui.externs.js
demo.compiled.js                       locales.js                             shaka-player.compiled.debug.externs.js shaka-player.compiled.map              shaka-player.ui.js
```

Comment (joeyparrish) [reply ajax/libs/shaka-player/package.json]: Nope.  Those are part of our demo app, and shouldn't be needed by other people's apps.

**Pull request merged**